### ISSUE TITLE
Some kind of weapon system

### ISSUE THREAD

**Issue by Polsaker** [tonyravioli/dongerdong#16]
It's time to overcomplicate the game:
We could add some kind of weapon/shield system, assigning a random kit to every player when they !accept

Quoting Mithadon:

>1 handed sword goes with shield so you have a chance to block but the sword is weaker than long sword, axe has a bigger variation from lowest to highest dmg dealt

**Comment by Polsaker** [comment]
><Nate848> Archer: high crit rate, low def
><Nate848> Mage: highest heal rate, lower def and hit
><Nate848> Knight: large hit and def but low to no hesl

**Comment by Mithadon** [comment]
Hit 18-35, crit x2 and 1/12 chance, instakill 1/50 chance, heal 25-44, praise 1/3 crit hit, crit health or nothing (that's how it is now)

Give players a 1 minute period during which to purchase items to use in battle, between weapons, armor and potions (usable during turn but which do not take a turn)
Swords: 2x smaller variation in the roll
Short sword, 0.8*base
Long sword, 1.3*base
Nail bat: normal variation, 1*base
Axe: 2x larger variation in the roll
One-handed axe, 0.8*base
Two-handed axe, 1.3*base

Leather armor, always reduce damage taken by 0.15*
Chainmail armor, always reduce damage taken by 0.30*
Shield, 1/2 chance of blocking for 0.35*base

Potions usable on turn but do not use up the turn:
Three potions which are revealed upon use:
Double damage for 1 turn potion (including doubling crit)
50 heal potion; 
Poison potion which is applied to the weapon and grants an additional 0.3*dmg dealt for the next 5 turns that player has (stronger than double damage potion, but with a delay)
Three potions which only display that a potion was used:
protection potion, stops hit (lasts until effect or 5 turns)
deflection potion, randomly deflects hit to another person, lasts like protection
friendship potion (heals yourself and your attacker upon being attacked)

**Comment by tonyravioli** [comment]
I like the weapons, not sure if I like the electricity/fire/ice thing.

**Comment by Polsaker** [comment]
Finished the weapon/shield thing, will leave this open because I like the idea of throwing potions and stuff

**Comment by tonyravioli** [comment]
One of the things I really enjoy(ed) about this game was that it can be quick - a little snack as opposed to a meal. The weapon/shield system seems to have taken that away. In the fights I've observed, it seems like there's really low damage happening (not as low as the 2-4HP thing we saw that one time, but still low). This makes the games last upwards of four or five minutes, which just turns tedious imo. Is there a way we can roll back to just before weapon implementation, and take a look at how we can do it differently?
